### PR TITLE
Update foreman_chef to 0.1.5

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: plugins/ruby-foreman-chef/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-chef (0.1.5-1) stable; urgency=medium
+
+  * Update foreman_chef to 0.1.5
+
+ -- root <root@chefubuntu.example.com>  Wed, 08 Jul 2015 07:21:42 +0000
```
Comment (domcleal) [inline plugins/ruby-foreman-chef/debian/changelog]: Who's that?

**Comment by domcleal** [comment]
:+1:

File: plugins/ruby-foreman-chef/debian/changelog (hunk 1 above)
Comment (ares) [inline plugins/ruby-foreman-chef/debian/changelog]: oops, new machine that was not configured correctly yet :-) thanks

**Comment by domcleal** [comment]
Merged to both deb/develop and 1.9, thanks @ares.  I'll leave you to build it.

**Pull request merged**